namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-prod/resources/hmpps-education-and-work-plan-api.tf

```diff
@@ -116,29 +116,3 @@ resource "kubernetes_secret" "education_and_work_plan_dlq" {
     sqs_queue_name = module.education_and_work_plan_domain_events_dead_letter_queue.sqs_name
   }
 }
-
-resource "kubernetes_secret" "ciag_domain_events_queue" {
```

Comment:
Likewise, the secrets for the education_and_work_plan_domain_events_queue and DLQ do not need setting up in the CIAG namespace as the CIAG API has been decommissioned (notice the namespace name is different to this namespace name. They were related but different services, and probably should not have been setup in this way anyway 🤷 )